tests/unit/src/flixel/system/replay/FlxReplayTest.hx

```diff
@@ -0,0 +1,69 @@
+package flixel.system.replay;
+
+import massive.munit.Assert;
+
+class FlxReplayTest
+{
+	var seed:Int;
+	var fgr:String;
+	var frameCount:Int;
+
+	@Before
+	function before()
+	{
+		seed = 123456789;
+		frameCount = 300;
+		fgr = seed + "\n";
+		var key:Int;
+		var x:Int;
+		var y:Int;
+		var possibleKeys = [8, 9, 13, 16, 17, 18, 20, 27, 32, 33, 34, 35, 36, 37, 38, 39, 40, 45, 46, 48, 49, 50, 51, 52, 53, 54, 55, 56, 57, 65, 66, 67, 68, 69, 70, 71, 72, 73, 74, 75, 76, 77, 78, 79, 80, 81, 82, 83, 84, 85, 86, 87, 88, 89, 90, 96, 97, 98, 99, 100, 101, 102, 103, 104, 105, 106, 107, 109, 110, 112, 113, 114, 115, 116, 117, 118, 119, 120, 121, 122, 123, 186, 187, 188, 189, 190, 191, 219, 221, 222, 301];
```

Comment:
Really? That's weird.. There should be a `FlxG.resetGame()` call after each test class at least.